KabakiAntony/todo-ui
# building the update password page

#### What does this PR do?
Create the update password page
#### Description of Task to be completed?
It starts out building the UI for the update or setting a new password page.
#### How should this be manually tested?
Clone this repo and then once you have followed the instructions on the readme on the develop
branch, you can only access this page if you have an email for a password reset that is going to
send the link for this page and a token to verify your identity
#### Screenshots (if appropriate)
![Screenshot 2022-02-04 at 11-51-05 Todos UI](https://user-images.githubusercontent.com/49396576/152500193-d954ea5a-c104-45ad-a53e-87a44acfa313.png)
